eo-phi-normalizer/src/Language/EO/Phi/Metrics/Collect.hs

```diff
@@ -38,34 +38,34 @@ collectMetrics a = execState (inspect a) defaultMetrics
 class Inspectable a where
   inspect :: a -> State Metrics ()
 
+count :: (a -> Bool) -> [a] -> Int
+count x = length . filter x
+
+countDataless :: (MonadState Metrics m) => [Binding] -> m ()
```

Comment:
This function requires documentation and a doctest.

Reply:
added in https://github.com/objectionary/normalizer/pull/142/commits/2fa396e05ce7e7e1e8311ee9b3483d7bde477a15